### ISSUE TITLE
Android: Tapping back (UI or navigation toolbar) after going to book (cookie browser) does nothing

### ISSUE THREAD

**Issue by yoasif** [Lusito/forget-me-not#257]
Steps to reproduce:

1. Go to add-on settings from menu
2. Tap book icon icon
3. Press back on Android navigation toolbar *or* use back button in Fenix UI 

What happens:

Nothing. User must tap "Back" to get out of this UI.

Expected result:

Tapping back takes me back to the main UI (like back).

**Comment by Lusito** [comment]
Indeed, the UI does not currently keep a history and thus can't handle the back button at all (anywhere). I'm gonna wait before fixing this though, since it's only a problem on android and the latest Firefox on Android removed support for most extensions, including mine.

Thanks for the report though, I didn't think of that scenario.
